func_adl_xAOD/common/event_collections.py

```diff
@@ -87,9 +87,11 @@ def get_collection(self, md: EventCollectionSpecification, call_node: ast.Call):
 
         # Replace it as the function that is going to get called.
         call_node.func = r  # type: ignore
-
         return call_node
 
+    def get_running_code_CPPCodeValue(self, cpv: cpp_ast.CPPCodeValue, md: EventCollectionSpecification):
```

Comment:
Need a comment after this method describing what is done - this is a method that has to be over ridden. Suggest something like: '''Using the `EventCollectionSpecification`, put the running code into the `CPPCodeValue`, along with anything else needed (like variable declarations).```